src/text/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { v, create } from '@dojo/framework/core/vdom';
+import theme from '../middleware/theme';
+
+import * as css from '../theme/default/text.m.css';
+
+export interface TextProperties {
+	size?: 'xs' | 's' | 'm' | 'l' | 'xl' | 'xxl';
+	inverse?: boolean;
+	truncated?: boolean;
+	as?: string;
```

Comment:
is it worth typing this to standard text nodes such as `span | div | h1-h6 | p` ?

Reply:
I thought about it, but it really could be any node, unless that's something we want to restrict...?

Reply:
I guess it's better to leave as is then, but might have been nice for auto-completion